OrPinhas_and_AvivShabtay_Project/OrPinhas_and_AvivShabtay/OrPinhas_and_AvivShabtay/Hotel.h

```diff
@@ -35,11 +35,11 @@ class Hotel
 		delete[] hotelName;
 	};
 
-//	friend ostream& operator<<(ostream& os, Hotel& hotel) const
-//	{
-//		os << 'List Of Crew For Current Shift:' << employee.crewList;
-//		return os;
-//	};
+	//	friend ostream& operator<<(ostream& os, Hotel& hotel) const
+	//	{
+	//		os << 'List Of Crew For Current Shift:' << employee.crewList;
+	//		return os;
+	//	};
 
 
 };
```

Comment:
If you want to prevent copy c'tor use: `Hotel(Hotel& hotle) = delete;`
The comment doesn't help to prevent it 😄